client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMetaMultiselect.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div>
+    <dp-label
+      :text="label"
+      :for="name" />
+    <ul
+      v-if="editable"
```

Comment:
I think the list should be shown if it is not editable and the multiselect if editable. Option names should be shown if they're defined and otherwise `-`.